taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/tasklistener/TaskanaTaskListener.java

```diff
@@ -190,6 +190,7 @@ private void prepareAndExecuteStatement(
       preparedStatement.setInt(4, initialRetries);
       preparedStatement.setTimestamp(5, eventCreationTimestamp);
       preparedStatement.setString(6, delegateTask.getId());
+      preparedStatement.setString(7, delegateTask.getProcessEngine().getName());
```

Comment:
How do we know, how long the name can get? Should be ensure at least that the name is not longer than 128 characters?

Reply:
What should happen, if it's longer than 128 chars? And should we give it more possible space to begin with?

Reply:
Good point. Actually we do not have a change to prevent failing in this case. All we could do is to throw a more explicit Exception in this case. Do you think it is worth it?